pkg/reconciler/route/route.go

```diff
@@ -344,7 +346,7 @@ func (c *Reconciler) tls(ctx context.Context, host string, r *v1alpha1.Route, tr
 		}
 		tls = append(tls, resources.MakeIngressTLS(cert, cert.Spec.DNSNames))
 	}
-	return tls, nil
+	return tls, acmeChallenges, nil
```

Comment:
Could we sort acmeChallenges to make sure acmeChallenges won't change if there is no content change?

Reply:
Yes, that makes sense.